models/staging/tpch/stg_tpch_line_items.sql

```diff
@@ -8,7 +8,7 @@ renamed as (
 
     select
     
-        {{ dbt_utils.surrogate_key(
+        {{ dbt_utils.generate_surrogate_key(
```

Comment:
doesn't seem to be an issue for this project but just in case, have in mind that default behaviour also changes for this function. If there are any snapshot or incremental models, you may want to add [this var](https://docs.getdbt.com/guides/migration/versions/upgrading-to-dbt-utils-v1.0#changes-to-surrogate_key) to the project.yml.

Reply:
Great catch Vicky! I will keep that in mind. This is a demo project and we run full-refresh all the time, but will add a doc note in the code to relate to your comment!